### DISCUSSION TITLE
closeAfterSelect as false on a Single choice Select field

### DISCUSSION THREAD

**Discussion by wiplano** [orchidjs/tom-select#165] [Q&A] [answered]
Hi!
Is it possible to keep the selection box always open even if my select field is of the "single" type?
"closeAfterSelect" as "false" does not work in this case.
I'm using tom-select for a color selection/search box. So, even if the user only has one option, he can still seeing the colors and choose another one.
Thx!

**Comment by oyejorge** [comment, marked as answer]
```closeAfterSelect=false``` now works for single selects in v2 [as of this commit](https://github.com/orchidjs/tom-select/commit/6f20eb553ff50b57482471e123404f1923bddf3e#diff-25a6634263c1b1f6fc4697a04e2b9904ea4b042a89af59dc93ec1f5d44848a26)

**Comment by wiplano** [comment]
I have implemented it. Very good!
Thank you!
